.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/auth@v2
+        with:
+          credentials_json: $GCLOUD_SERVICE_KEY
+
+      - uses: google-github-actions/setup-gcloud@v2
+        with:
+          project_id: $GOOGLE_PROJECT_ID
```

Comment:
I think you can just

```
- run: gcloud --quiet config set compute/zone ${GOOGLE_COMPUTE_ZONE}
```

here, right?  It doesn't look like there is an option like with the project_id, but it is just running the gcloud command under the hood anyways: https://github.com/google-github-actions/setup-cloud-sdk/blob/7458dd222121d80ff6b491103733e25cbadc0be7/src/index.ts#L236